packages/nextjs/components/scaffold-eth/Address/Address.tsx

```diff
@@ -177,7 +177,7 @@ export const Address = ({
             </AddressLinkWrapper>
           </span>
           <AddressCopyIcon
-            className={`ml-1 text-sky-600 ${copyIconSizeMap[addressSize]} cursor-pointer`}
+            className={`ml-1 text-base-content ${copyIconSizeMap[addressSize]} cursor-pointer`}
```

Comment:
Maybe we completely remove text-color className from this. 

Then it will follow the color which people set in outer container, otherwise it will still remain hardcode 

![image](https://github.com/user-attachments/assets/bf272c29-d0a8-4b09-a36d-66cf70a0b517)

Reply:
good point, changed for other copy icons too
[155418f](https://github.com/scaffold-eth/scaffold-eth-2/pull/991/commits/155418fd7084cb85214c743b4615474e7be276ac)